src/librustc_lint/builtin.rs

```diff
@@ -577,74 +577,71 @@ impl LateLintPass for MissingDebugImplementations {
 declare_lint! {
     DEPRECATED,
     Warn,
-    "detects use of #[rustc_deprecated] items"
+    "detects use of `#[deprecated]` or `#[rustc_deprecated]` items"
```

Comment:
The existence of `#[rustc_deprecated]` is an implementation detail. Can we change this string to 'detects use of deprecated items'?